### PR TITLE
Add leases for ppc64le and s390x multiarch

### PR DESCRIPTION
**Pull request by werled**
Analogous to the changes in #3358 (combines #3467, #3468)

### CODE REVIEW

File: pkg/api/types.go [hunk 1]
```diff
@@ -1386,7 +1390,9 @@ func (p ClusterProfile) ClusterType() string {
 		return string(CloudGCP)
 	case
 		ClusterProfileIBMCloud,
-		ClusterProfileIBMCloudQE:
+		ClusterProfileIBMCloudQE,
+		ClusterProfileIBMCloudMultiPpc64le, // TODO: Check if the return value should be "ibmcloud-multi-ppc64le" instead of "ibmcloud"
```
Comment (jaypoulz) [inline pkg/api/types.go:1394]: This is ultimately a billing question. Let's find out who owns the wallet for the IBM Cloud account(s)

Comment (jaypoulz) [reply pkg/api/types.go:1394]: I suspect we will want "ibmcloud-multi-ppc64le" and "ibmcloud-multi-s390x" so that we can have our own cluster profiles. :)

Comment (jaypoulz) [reply pkg/api/types.go:1394]: (which in turn will specify the leases).

Comment (werled) [reply pkg/api/types.go]: Thanks for the comments, updated :).

**Comment by werled** [comment]
As far as I understand, the failing check has nothing to do with the changes in the PR, but currently doesn't pass for all PRs.

**Pull request merged**